src/main.cpp

```diff
@@ -115,6 +117,19 @@ int main ( int argc, char** argv )
     QString      strClientName               = "";
     QString      strJsonRpcSecretFileName    = "";
 
+#if defined( HEADLESS ) || defined( SERVER_ONLY )
+    Q_UNUSED ( bStartMinimized )
+    Q_UNUSED ( bUseTranslation )
+    Q_UNUSED ( bShowComplRegConnList )
+    Q_UNUSED ( bShowAnalyzerConsole )
+    Q_UNUSED ( bMuteStream )
+#endif
+#if defined( SERVER_ONLY )
+    Q_UNUSED ( bMuteMeInPersonalMix )
+    Q_UNUSED ( bNoAutoJackConnect )
+    Q_UNUSED ( bCustomPortNumberGiven )
+#endif
+
```

Comment:
This seems related to #2574, not to JSON-RPC.

Reply:
I was tidying things up.

Reply:
I‘d still like it as a separate commit. No need to be in a separate PR

Reply:
Done